docs/resources/cluster.md

```diff
@@ -1,100 +1,334 @@
+    volume_properties = "gp3"
+    size              = "4 Gi"
+  }
+
+  pg_type            = "epas"
+  pg_version         = "14"
+  private_networking = false
+  cloud_provider     = "aws"
+  region             = "us-east-1"
+}
+
+output "password" {
+  sensitive = true
+  value     = resource.biganimal_cluster.eha_cluster.password
+}
+
+output "connection_uri" {
+  value = resource.biganimal_cluster.eha_cluster.connection_uri
+}
+```
 
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Required
 
-- `cloud_provider` (String) Cloud Provider
-- `cluster_architecture` (Block List, Min: 1) Cluster Architecture (see [below for nested schema](#nestedblock--cluster_architecture))
+- `cloud_provider` (String) Cloud Provider. e.g. "aws" or "azure".
+- `cluster_architecture` (Block List, Min: 1) Cluster Architecture. See https://www.enterprisedb.com/docs/biganimal/latest/overview/02_high_availability/ for details. (see [below for nested schema](#nestedblock--cluster_architecture))
 - `cluster_name` (String) Name of the cluster.
-- `instance_type` (String) Instance Type
-- `password` (String, Sensitive) Password
-- `pg_type` (String) Postgres type
-- `pg_version` (String) Postgres Version
-- `region` (String) Region
-- `storage` (Block List, Min: 1) Storage (see [below for nested schema](#nestedblock--storage))
+- `instance_type` (String) Instance Type. e.g. "azure:Standard_D2s_v3", "aws:c5.large".
+- `password` (String, Sensitive) Password for the user edb_admin. It must be 12 characters or more.
+- `pg_type` (String) Postgres type. e.g. "epas", "pgextended" or "postgres".
+- `pg_version` (String) Postgres version. See https://www.enterprisedb.com/docs/biganimal/latest/overview/05_database_version_policy/#supported-postgres-types-and-versions for supported Postgres types and versions.
+- `region` (String) Region to deploy the cluster. See https://www.enterprisedb.com/docs/biganimal/latest/overview/03a_region_support/ for supported regions.
+- `storage` (Block List, Min: 1) Storage. (see [below for nested schema](#nestedblock--storage))
 
 ### Optional
 
-- `allowed_ip_ranges` (Block List) Allowed IP ranges (see [below for nested schema](#nestedblock--allowed_ip_ranges))
-- `backup_retention_period` (String) Backup Retention Period.
-- `pg_config` (Block List) Instance Type (see [below for nested schema](#nestedblock--pg_config))
-- `private_networking` (Boolean) Is private networking enabled
-- `read_only_connections` (Boolean) Is read only connections enabled
+- `allowed_ip_ranges` (Block List) Allowed IP ranges. (see [below for nested schema](#nestedblock--allowed_ip_ranges))
+- `backup_retention_period` (String) Backup Retention Period. e.g. "7d", "2w" or "3m".
+- `pg_config` (Block List) Database Configuration Parameters. See https://www.enterprisedb.com/docs/biganimal/latest/using_cluster/03_modifying_your_cluster/05_db_configuration_parameters/ for details. (see [below for nested schema](#nestedblock--pg_config))
```

Comment:
```suggestion
- `pg_config` (Block List) Database Configuration Parameters. See [Modifying database configuration parameters ](https://www.enterprisedb.com/docs/biganimal/latest/using_cluster/03_modifying_your_cluster/05_db_configuration_parameters/) for details. For schema details, see [nested schema](#nestedblock--pg_config).
```

Reply:
Clickable link is added at https://github.com/EnterpriseDB/terraform-provider-biganimal/commit/0aea7392df32d1d3b7fcb6e35524aa585fa8a660